doc/rst/technotes/gpu.rst

```diff
@@ -5,64 +5,53 @@
 GPU Programming
 ===============
 
-Chapel includes preliminary work to target NVidia GPUs using CUDA. This work is
-under active development and has not yet been tested under a wide variety of
-environments. We have tested it on systems with NVidia Tesla P100 GPUs and CUDA
-11.0 and a system with NVidia Ampere A100 GPUs with CUDA 11.6. The current
-implementation will only apply to certain ``forall`` and ``foreach`` loops.
-
-We also require ``LLVM`` to be used as Chapel's backend compiler (i.e.
-``CHPL_LLVM`` must be set to ``system`` or ``bundled``). For more information
-about these settings see :ref:`Optional Settings <readme-chplenv>`.
+Chapel includes preliminary work to target NVidia GPUs by generating and
+packing PTX assembly and linking against and using the CUDA driver API at
+runtime. This work is under active development and has not yet been tested
+under a wide variety of environments. We have tested it on systems with NVidia
+Tesla P100s using CUDA 11.0 and on a system with NVidia Ampere A100 with CUDA
+11.6. The current implementation will generate CUDA kernel code (PTX assembly)
+for certain ``forall`` and ``foreach`` loops and these kernels will be launched
+onto a GPU when the current locale (e.g. ``here``) is assigned to a special
+(sub)locale representing the GPU.
+
+For more information about what loops are eligible for GPU execution see the
+`Overview`_ section.  For more information about what is supported see the
+requirements and `Requirements and Limitations`_ section.  To see an example
+program written in Chapel that will execute on a GPU see the code listing in
+the `Example`_ section.  For more information about sepcific features related
+to GPU support see the subsections under `GPU Support Features`_.  Additional
+information about GPU Support can be found in the "Ongoing Efforts" slide decks
+of our `release notes <https://chapel-lang.org/releaseNotes.html>`_; however,
+be aware that information presented in releease notes for prior releases may be
+out-of-date.
+
+.. contents::
 
 Overview
 --------
 
 To deploy code to a GPU, put the relevant code in an ``on`` statement targeting
 a GPU sublocale (i.e. ``here.gpus[0]``).
 
-Any arrays that are declared by tasks executing on a GPU sublocale will be
-allocated into unified memory and will be accessible on the GPU. Chapel will
-launch CUDA kernels for all eligible loops (instead of running the loop on the
-CPU) that are encountered by tasks executing on a GPU sublocale. Loops are
-eligible when:
+Any arrays that are declared by tasks executing on a GPU sublocale will, when
```

Comment:
This kind of makes it sound like the default memory strategy is something bigger, and that the use of unified memory is just one of its aspects. I think it would sound less like that if this just said "will, by default, be allocated into unified memory ..."

Reply:
Changed to:

"Any arrays that are declared by tasks executing on a GPU sublocale will, by
default, be allocated into unified memory and be accessible on the GPU (see the
`Memory Strategies`_ subsection for more information about alternate memory
strategies)."